setup.cfg

```diff
@@ -1,6 +1,7 @@
 [flake8]
-max-line-length=100
+max-line-length=88
 exclude = docs,.venv
+ignore = E203, E501, W503, F811, F401
```

Comment:
Just curious, what do we need F401 ("module not imported") for?

Reply:
It didn't like how some of the decorators were used in bionic